src/main/java/frc/robot/Constants.java

```diff
@@ -135,6 +135,27 @@ public static final class OIConstants {
     public static final double kArmManualSpeed = 0.01;
   }
 
+  public static final class ControlContants {
```

Comment:
Generally you want to have the constants as what the actual buttons are instead of what commands it's going to run. see this code for how they implemented it
https://github.com/triplestrange/RobotCode2023/blob/main/RobotCode2023/src/main/java/frc/robot/Constants.java
names of buttons could be better lol

Reply:
oooh that makes sense. Ill fix soon